### ISSUE TITLE
CS2:  Increase Radius of Chained Attacks

### ISSUE THREAD

**Issue by XanaduCSIV** [AdmiralCurtiss/SenPatcher#64]
I was wondering if it was possible to put a fix into the game to increase the radius of chained battle attacks on the field maps?  I love chained battles, but when they introduced this in CS2, I felt the range was so tight that it was challenging to get the fight to chain 2 or 3 battles when grouping sprites all in one small area.  If that was increased, then it could be more fun to chain more fights together.

I'm intending to log this as a "game enhancement".  Forgive me if I'm not posting this right.

**Comment by AdmiralCurtiss** [comment]
Yes this is the right place. Can't guarantee this will ever happen, but noted anyway.
